api/interservice/authz/v2/policy_test.go

```diff
@@ -134,31 +134,11 @@ func TestValidationCreatePolicy(t *testing.T) {
 			Statements: []*v2.Statement{&validStatement},
 		},
 		// projects
-		"whitespace projects list": &v2.CreatePolicyReq{
```

Comment:
these are all pushed into server?

Reply:
Several that were about validation are removed as they are now the equivalent of a "not found"-- instead of duplicating more invalid things that would result in "not found", I removed them.